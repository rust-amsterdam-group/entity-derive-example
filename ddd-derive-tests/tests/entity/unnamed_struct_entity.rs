use ddd_derive::Entity;

#[derive(Entity)]
struct Person(i32, String);

fn main() {}