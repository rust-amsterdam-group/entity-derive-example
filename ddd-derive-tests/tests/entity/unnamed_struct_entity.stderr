error: Entity cannot be derived from Struct with unnamed fields

  = note: Entity may only be derived from a Struct with named fields
  = help: Try changing the item to a Struct with named fields

 --> $DIR/unnamed_struct_entity.rs:4:1
  |
4 | struct Person(i32, String);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
